imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -25,16 +25,23 @@
     "Magnetic Fields (space)",
 )
 
-mag_l1a_attrs = GlobalDataLevelAttrs(
-    # TODO: data type should include "norm" and "burst" L1A-norm>Level-1A-normal-rate
-    "L1A>Level-1A",
+mag_l1a_norm_raw_attrs = GlobalDataLevelAttrs(
+    "L1A-raw-norm>Level-1A-raw-normal-rate",
```

Comment:
Here, Andriy said that this data_type should be datalevel_descriptor. I think it's ok to have `-` in descriptor. This is what I think he may want. @bryan-harter, what do you think?
```suggestion
    "L1A-raw-norm>l1a_raw-normal-rate",
```

Reply:
I think this was a direct note from an email he sent me about descriptors - but I'd appreciate Bryan's thoughts

Reply:
Ah..interesting. My notes said differently. Good catch.

Reply:
Sorry, better late than never? I do go back and look at reviews eventually lol.  

I'm pretty sure it can just be "L1A-raw-norm>Level-1A Raw Normal Rate".  No need for hyphens or underscores on the other side of the ">".

Reply:
Thanks! I'll update in my next mag PR!